### ISSUE TITLE
Flickering effect on background tab "Scripts"

### ISSUE THREAD

**Issue by Cortek29** [jmoenig/Snap--Build-Your-Own-Blocks#1808]
Hi,
I have sensitive eyes, and this background is very unpleasant for me...
I found a solution for this flickering
In file "gui.js" on line 183, the function "IDE_Morph.prototype.scriptsTexture"
i modify this : 
"IDE_Morph.prototype.scriptsTexture = function () {
    var pic = newCanvas(new Point(100, 100)), // bigger scales faster
        ctx = pic.getContext('2d'),
        i;
    for (i = 0; i < 100; i += **10**) { //instead 4
        ctx.fillStyle = this.frameColor.toString();
        ctx.fillRect(i, 0, 1, 100);
        ctx.fillStyle = this.groupColor.lighter(2).toString();
        ctx.fillRect(i + 1, 0, 1, 100);
        ctx.fillRect(i + **20**, 0, 1, 100); //instead 3
        ctx.fillStyle = this.groupColor.toString();
        ctx.fillRect(i + 2, 0, 1, 100);
    }
    return pic;
};"
If you can add an option to change this as a parameter...

**Comment by DavidGasku** [comment]
Yes please! While it wasn't much noticeable in desktops, we've purchased new chromebooks this year, and it's really annoying.

**Comment by cycomachead** [comment]
As an imperfect work around, you might try flat design in the settings. It's got a gray-ish background.
